web/ui/static/js/graph.js

```diff
@@ -1,16 +1,18 @@
 var Prometheus = Prometheus || {};
-var graphs = [];
 var graphTemplate;
 
 var SECOND = 1000;
 
 Handlebars.registerHelper('pathPrefix', function() { return PATH_PREFIX; });
 
-Prometheus.Graph = function(element, options) {
+Prometheus.Graph = function(element, options, changeHandler, removeHandler) {
```

Comment:
This is fixed at https://github.com/prometheus/prometheus/pull/1872/commits/66c2d3dab32d7a989f2e21fda7fa9dffb516786a